setup.py

```diff
@@ -0,0 +1,25 @@
+"""Setup file for PyRCCA"""
+
+from setuptools import setup, find_packages
+from pathlib import Path
+
+readme_path = Path(__file__).parent / 'README.md'
+with open(readme_path, encoding='utf-8') as f:
+    long_description = f.read()
+
+setup(
+    name='pyrcca',
+    version=0.1,
+    author='Bilenko et al',
+    packages=find_packages(),
+    url = 'https://github.com/gallantlab/pyrcca',
+    license='MIT',
```

Comment:
I don't think it is a MIT license, though I don't know how to call it ...

Reply:
It seems to be called "academic license" (cf [origin](https://ipira.berkeley.edu/software-copyright-notice-and-disclaimer) and [FAQ](https://ipira.berkeley.edu/FAQs)).

> Another license commonly used at Berkeley was developed by the UC legal office and is [colloquially] known as the academic license.

Reply:
Maybe "Free for non-commercial use" ? ([List of valid PyPI licenses](https://autopilot-docs.readthedocs.io/en/latest/license_list.html))